test/controllers/reviews_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
```

Comment:
We are usually using RSpec+FactoryGirl in our projects, and i think you'll benefit from learning that. Talk to Yaniv about it.